dpgen2/entrypoint/args.py

```diff
@@ -232,6 +232,7 @@ def input_args():
     doc_type_map = 'The type map. e.g. ["Al", "Mg"]. Al and Mg will have type 0 and 1, respectively.'
     doc_mass_map = "The mass map. e.g. [27., 24.]. Al and Mg will be set with mass 27. and 24. amu, respectively."
     doc_mixed_type = "Use `deepmd/npy/mixed` format for storing training data."
+    doc_finetune_mode = "Finetune the pretrained model in the first iteration."
```

Comment:
Please tell the user, in detail, the available options and their meanings for this argument.